parser/src/main/scala/aqua/parser/head/FromExpr.scala

```diff
@@ -27,4 +27,10 @@ object FromExpr {
 
   def importFrom[F[_]: LiftParser: Comonad]: P[NonEmptyList[NameOrAbAs[F]]] =
       comma[NameOrAbAs[F]](nameOrAbAs[F]) <* ` ` <* `from`
+
+  def show[F[_]](ne: NonEmptyList[NameOrAbAs[F]]): String =
+    ne.toList.map(_.fold(
+      non => non._1.value + non._2.map(_.value).fold("")(" as "+_),
```

Comment:
We should refactor it one day... e.g. when we switch to tagged types instead of Name and Ability.